heft-plugins/heft-jest-plugin/package.json

```diff
@@ -12,10 +12,10 @@
   "types": "dist/heft-jest-plugin.d.ts",
   "license": "MIT",
   "scripts": {
-    "build": "heft build --clean",
-    "start": "heft test --clean --watch",
-    "_phase:build": "heft build --clean",
-    "_phase:test": "heft test --no-build"
+    "build": "node ./node_modules/@rushstack/heft-legacy/bin/heft --unmanaged build --clean",
```

Comment:
It could, yes. The main reason why I avoided doing that was to align all the heft plugins included in the `heft-node-rig` to use a cyclic dependency. This way I could get all the features that the rig provides (build + lint + test + api-extractor). In fact, some linting issues were found when I changed it to build this way, which kinda made me feel like this is in fact the more "correct" way of doing it for this project. Once everything is merged and published, I want to do a second pass to update the build to use the newer version of Heft, but I don't think we need it to be using the workspace version.

Reply:
So, to confirm, the plan is to:

1. Check this in with the heft-legacy thing
2. Publish everything
3. For the plugins included in `heft-node-rig`, use a cyclic dependency on the published version of the rig

> I don't think we need it to be using the workspace version.

It probably should be using the workspace version so any API changes to Heft get picked up.

Reply:
To clarify, we will be using the workspace version for the API interactions, and will have an optional peer dependency on the workspace version or newer. We're only using the non-workspace version for the project build, which I think is a fair compromise.

The heft-legacy thing will be updated to the newer version of Heft (though not the workspace version) after the initial publish.